src/Augurk/templates/configuration.html

```diff
@@ -112,6 +112,46 @@ <h2>Customization &amp; Configuration</h2>
         </div>
     </div>
 
+    <div class="panel panel-default">
+        <div class="panel-heading">Import &amp; Export</div>
+        <div class="panel-body">
+            <div class="row" style="margin-top: 10px;">
+                <div class="col-md-6">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">Import</div>
+                        <div class="panel-body">
+                            <p>
+                                Choose a file to import and then click the import button to restore the data from the
+                                selected file into this instance of Augurk.
+                            </p>
+                            <p>
+                                <strong>Note:</strong> This will <strong>overwrite</strong> any existing <strong>products,
+                                groups, features and tags</strong> with data from the imported file.
+                            </p>
+                            <input type="file" id="importFile" accept=".bak" />
+                            <button type="button" class="btn btn-danger pull-right" ng-click="import()">Import</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">Export</div>
+                        <div class="panel-body">
+                            <p>
+                                Use the export button to download an export of all the data contained within Augurk as
+                                a file. This file could be used to import the same data into another instance of Augurk.
+                            </p>
+                            <p>
+                                <strong>Note:</strong> This includes <strong>products, groups, features and tags.</strong>
+                            </p>
+                            <button type="button" class="btn btn-success pull-right" ng-click="export()">Export</button>
```

Comment:
I think this button should be blue, similar to all other buttons on this page.
The only button that should have a different color is de Import because the uses does something irrevocable there.